src/style.css

```
@@ -1,3 +1,87 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500px;
+  font-size: 12px;
+}
+
+.header {
```

Need to add it for fix scroll
```suggestion
}

* {
  box-sizing: border-box;
}

.header {
```